[2018-09-20T04:22:51.790] [ERROR] err - { Error: Not Found
    at D:\ManyKitHost\trunk\manykit\app.js:308:15
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:764:28)
    at D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:772:23
    at FSReqWrap.oncomplete (fs.js:152:21) status: 404 }
[2018-09-20T11:44:25.390] [ERROR] err - TypeError: This library (validator.js) validates strings only
    at assertString (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\util\assertString.js:11:11)
    at ltrim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\ltrim.js:15:30)
    at Object.trim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\trim.js:19:51)
    at login (D:\ManyKitHost\trunk\manykit\routes\func.js:315:38)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:736:16)
    at onstat (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:725:14)
[2018-09-20T11:52:19.991] [ERROR] err - ReferenceError: password is not defined
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:544:16)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T11:52:46.629] [ERROR] err - ReferenceError: password is not defined
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:544:16)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T12:51:37.631] [ERROR] err - TypeError: This library (validator.js) validates strings only
    at assertString (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\util\assertString.js:11:11)
    at ltrim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\ltrim.js:15:30)
    at Object.trim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\trim.js:19:51)
    at signup (D:\ManyKitHost\trunk\manykit\routes\func.js:123:34)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:736:16)
    at onstat (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:725:14)
[2018-09-20T14:44:10.215] [ERROR] err - TypeError: This library (validator.js) validates strings only
    at assertString (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\util\assertString.js:11:11)
    at ltrim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\ltrim.js:15:30)
    at Object.trim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\trim.js:19:51)
    at snaplogin (D:\ManyKitHost\trunk\manykit\routes\func.js:393:38)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:736:16)
    at onstat (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:725:14)
[2018-09-20T16:00:13.883] [ERROR] err - Error: Can't set headers after they are sent.
    at validateHeader (_http_outgoing.js:494:11)
    at ServerResponse.setHeader (_http_outgoing.js:501:3)
    at ServerResponse.header (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:730:10)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:256:15)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:158:21)
    at Object.success (D:\ManyKitHost\trunk\manykit\common\responseResult.js:4:24)
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:244:48)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T16:07:49.042] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at session (D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:454:7)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
[2018-09-20T16:10:20.620] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:14:02.240] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:155:25)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:14:40.587] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:14:45.089] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:14:46.180] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:17:08.571] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:17:29.830] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:17:33.056] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:20:53.917] [ERROR] err - SyntaxError: Identifier 'codeSql' has already been declared
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:89:10)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
    at Command.callback (D:\ManyKitHost\trunk\manykit\node_modules\connect-redis\lib\connect-redis.js:167:14)
    at normal_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:726:21)
    at RedisClient.return_reply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:824:9)
    at JavascriptRedisParser.returnReply (D:\ManyKitHost\trunk\manykit\node_modules\redis\index.js:192:18)
[2018-09-20T16:42:42.153] [ERROR] err - TypeError: This library (validator.js) validates strings only
    at assertString (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\util\assertString.js:11:11)
    at ltrim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\ltrim.js:15:30)
    at Object.trim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\trim.js:19:51)
    at login (D:\ManyKitHost\trunk\manykit\routes\func.js:315:38)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at serveStatic (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:75:16)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at D:\ManyKitHost\trunk\manykit\node_modules\express-session\index.js:489:7
[2018-09-20T16:44:32.409] [ERROR] err - ReferenceError: qqname is not defined
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:363:28)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T16:47:07.313] [ERROR] err - ReferenceError: email is not defined
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:363:53)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T17:22:43.005] [ERROR] err - Error: Can't set headers after they are sent.
    at validateHeader (_http_outgoing.js:494:11)
    at ServerResponse.setHeader (_http_outgoing.js:501:3)
    at ServerResponse.header (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:730:10)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:256:15)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:158:21)
    at Object.success (D:\ManyKitHost\trunk\manykit\common\responseResult.js:4:24)
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:244:48)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T17:40:37.345] [ERROR] err - Error: Can't set headers after they are sent.
    at validateHeader (_http_outgoing.js:494:11)
    at ServerResponse.setHeader (_http_outgoing.js:501:3)
    at ServerResponse.header (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:730:10)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:256:15)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:158:21)
    at Object.success (D:\ManyKitHost\trunk\manykit\common\responseResult.js:4:24)
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:244:48)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T17:59:26.544] [ERROR] err - { Error: Not Found
    at D:\ManyKitHost\trunk\manykit\app.js:308:15
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:736:16)
    at onstat (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:725:14)
    at FSReqWrap.oncomplete (fs.js:152:21) status: 404 }
[2018-09-20T18:00:55.595] [ERROR] err - Error: Can't set headers after they are sent.
    at validateHeader (_http_outgoing.js:494:11)
    at ServerResponse.setHeader (_http_outgoing.js:501:3)
    at ServerResponse.header (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:730:10)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:256:15)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:158:21)
    at Object.success (D:\ManyKitHost\trunk\manykit\common\responseResult.js:4:24)
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:244:48)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T18:11:04.055] [ERROR] err - Error: Can't set headers after they are sent.
    at validateHeader (_http_outgoing.js:494:11)
    at ServerResponse.setHeader (_http_outgoing.js:501:3)
    at ServerResponse.header (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:730:10)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:256:15)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:158:21)
    at Object.success (D:\ManyKitHost\trunk\manykit\common\responseResult.js:4:24)
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:244:48)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[2018-09-20T18:57:31.735] [ERROR] err - TypeError: This library (validator.js) validates strings only
    at assertString (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\util\assertString.js:11:11)
    at ltrim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\ltrim.js:15:30)
    at Object.trim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\trim.js:19:51)
    at snaplogin (D:\ManyKitHost\trunk\manykit\routes\func.js:422:38)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:736:16)
    at onstat (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:725:14)
[2018-09-20T21:04:18.387] [ERROR] err - TypeError: This library (validator.js) validates strings only
    at assertString (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\util\assertString.js:11:11)
    at ltrim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\ltrim.js:15:30)
    at Object.trim (D:\ManyKitHost\trunk\manykit\node_modules\validator\lib\trim.js:19:51)
    at login (D:\ManyKitHost\trunk\manykit\routes\func.js:315:38)
    at Layer.handle [as handle_request] (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:317:13)
    at D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:335:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:116:13)
    at SendStream.emit (events.js:211:7)
    at SendStream.error (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:275:17)
    at SendStream.onStatError (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:426:12)
    at next (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:736:16)
    at onstat (D:\ManyKitHost\trunk\manykit\node_modules\send\index.js:725:14)
[2018-09-20T21:40:20.246] [ERROR] err - Error: Can't set headers after they are sent.
    at validateHeader (_http_outgoing.js:494:11)
    at ServerResponse.setHeader (_http_outgoing.js:501:3)
    at ServerResponse.header (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:730:10)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:256:15)
    at ServerResponse.send (D:\ManyKitHost\trunk\manykit\node_modules\express\lib\response.js:158:21)
    at Object.success (D:\ManyKitHost\trunk\manykit\common\responseResult.js:4:24)
    at publicSql.then (D:\ManyKitHost\trunk\manykit\routes\func.js:244:48)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
